src/main/scala/wdl4s/WdlNamespace.scala

```diff
@@ -143,76 +147,82 @@ case class WdlNamespaceWithWorkflow(importedAs: Option[String],
   */
 object WdlNamespace {
   def load(wdlFile: Path): WdlNamespace = {
-    load(readFile(wdlFile), wdlFile.toString, localImportResolver, None)
+    load(readFile(wdlFile), wdlFile.toString, fileResolver)
   }
 
   def load(wdlFile: Path, importResolver: ImportResolver): WdlNamespace = {
-    load(readFile(wdlFile), wdlFile.toString, importResolver, None)
+    load(readFile(wdlFile), wdlFile.toString, importResolver)
   }
 
   def load(wdlSource: WdlSource): WdlNamespace = {
-    load(wdlSource, "string", localImportResolver, None)
+    load(wdlSource, "string",  Seq(fileResolver), None)
+  }
+
+  def load(wdlSource: WdlSource, importResolver:ImportResolver): WdlNamespace = {
+    load(wdlSource, "string",  Seq(importResolver), None)
   }
 
-  def load(wdlSource: WdlSource, importResolver: ImportResolver): WdlNamespace = {
+  def load(wdlSource: WdlSource, importResolver: Seq[ImportResolver]): WdlNamespace = {
     load(wdlSource, "string", importResolver, None)
   }
 
   def load(wdlSource: WdlSource, resource: String): WdlNamespace = {
-    load(wdlSource, resource, localImportResolver, None)
+    load(wdlSource, resource, Seq(fileResolver), None)
   }
 
   def load(wdlSource: WdlSource, resource: String, importResolver: ImportResolver): WdlNamespace = {
-    load(wdlSource, resource, importResolver, None)
+    load(wdlSource, resource, Seq(importResolver), None)
   }
 
-  private def load(wdlSource: WdlSource, resource: String, importResolver: ImportResolver, importedAs: Option[String]): WdlNamespace = {
+  private def load(wdlSource: WdlSource, resource: String, importResolver: Seq[ImportResolver], importedAs: Option[String]): WdlNamespace = {
```

Comment:
`importedAs` might as well be a `String`. We should always have something to import it as now